cv19ResSupportV3/Startup.cs

```diff
@@ -128,9 +128,13 @@ public static void ConfigureServices(IServiceCollection services)
             RegisterUseCases(services);
         }
 
-        private static void ConfigureDbContext(IServiceCollection services)
+        private void ConfigureDbContext(IServiceCollection services)
         {
+#if DEBUG
+            var connectionString = Configuration.GetConnectionString("DevDocker");
+#else
             var connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
+#endif
```

Comment:
Nice! Didn't expect to see the preprocessor directives.

Reply:
Though, I'm confused as heck as to how it's working, considering the make serve is using docker compose to launch the service & the docker compose of the application's container is not given a "DevDocker" value. :thinking: 

![image](https://user-images.githubusercontent.com/43747286/136194798-364c86c0-35ec-4601-ae72-e8cb699cd425.png)

Reply:
DevDocker is the name of the connection string which I put in appSettings.Development.json. It's plausible this connection string could change, but it's at least stored in the project now rather than a separate environment variable on the developer's machine.